app/dashboard/templates/profiles/tab_follow.html

```diff
@@ -0,0 +1,145 @@
+        <div style="height: fit-content">
+          <a class="btn btn-sm flex-grow-1 mr-5 font-smaller-5 position-relative quick-link btn-outline-gc-blue" {% if not user.org.email %} hidden {% endif %} href="mailto:{{user.org.email}}"  data-placement="bottom" data-toggle="tooltip" data-html="true" title="@{{ user.org.handle }}'s Email">
+            <i class="far fa-envelope"></i>
+          </a>
+        </div>
+        <div class="d-flex align-items-center" style="height: fit-content">
+          {%  with followers=user.mutual_follower %}
+            {% if followers|length == 1 %}
+                <div style="width: 60px">
+                    <img class="rounded-circle" style="height: 25px; width: 25px;" src="/dynamic/avatar/{{ followers.0.profile.handle }}" alt="">
+                </div>
+                <p class="text-muted font-smaller-4 my-0">Followed by {{ followers.0.profile.handle }}</p>
+            {% endif %}
+            {% if followers|length == 2 %}
+                <div style="width: 60px" class="d-flex flex-row align-items-center">
+                    <img class="rounded-circle" style="height: 25px; width: 25px; margin-right: -12px " src="/dynamic/avatar/{{ followers.0.profile.handle }}" alt="">
+                    <img class="rounded-circle" style="height: 25px; width: 25px;" src="/dynamic/avatar/{{ followers.1.profile.handle }}" alt="">
+                </div>
+                <p class="text-muted font-smaller-4 my-0">Followed by {{ followers.0.profile.handle  }} and {{ followers.1.profile.handle }}</p>
+            {% endif %}
+            {% if followers|length >= 3 %}
+                <div style="width: 60px" class="d-flex flex-row align-items-center">
+                    <img class="rounded-circle" style="height: 25px; width: 25px; margin-right: -12px" src="/dynamic/avatar/{{ followers.0.profile.handle }}" alt="">
+                    <img class="rounded-circle" style="height: 25px; width: 25px; margin-right: -12px" src="/dynamic/avatar/{{ followers.1.profile.handle }}" alt="">
+                    <img class="rounded-circle" style="height: 25px; width: 25px;" src="/dynamic/avatar/{{ followers.2.profile.handle }}" alt="">
+                </div>
+                <p class="text-muted font-smaller-4 my-0">Followed by {{ followers.0.profile.handle  }}, {{ followers.1.profile.handle }} and {{ followers|length|add:-2}} others you follow</p>
+            {% endif %}
+          {% endwith %}
+        </div>
+
+        </div>
+        <div class="col-12 col-md-3 col-lg-3 font-caption my-auto text-right">
+          <button class="btn btn-sm m-4 px-4 font-caption py-1 font-weight-bold btn-outline-gc-green" style="border-width: 2px !important;" data-tribe="{{user.org.handle}}">Following</button>
+        </div>
+      </div>
+      {% endfor %}
+</div>
+
+
+  <script>
+    setInterval(function(){
+      if($ && $('.load-more').length && $(".load-more").isInViewport()){
+        $('.load-more').click();
+      }
+    }, 1000);
```

Comment:
how this work?  doesn't seems to be any ajax request handling this and if checking each second to click the button? Or I am missing something?

Reply:
borrowed from a different tab, I think tab_kudos.html. Possibly didn't connect it, will check that.

Reply:
done